dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -41,9 +41,9 @@ ERROR_UNINSTALL_FIX_INVALID_META_DATA.explanation=Internal error detected proces
 ERROR_UNINSTALL_FIX_INVALID_META_DATA.useraction=See the problem determination information on the WebSphere Application Server Support web page: http://www.ibm.com/software/webservers/appserv/was/support.
 
 # Following messages for InstallException
-ERROR_FAILED_TO_RESOLVE_FEATURES=CWWKF1203E: Unable to obtain the following features: {0}. Ensure that the features are valid.
+ERROR_FAILED_TO_RESOLVE_FEATURES=CWWKF1203E: Unable to obtain the following features: {0}. Ensure that the features are valid for {1}.
```

Comment:
We should use introduce a separate message for Open Liberty.  The original message can remain the same as before (don’t need to mention the edition for WebSphere Liberty).  The new message can be like:
`Unable to obtain the following features: {0}. Ensure that the features are valid for Open Liberty. 
explanation=The installer could not retrieve the specified features from the connected repositories. 
useraction=If specified features exist, ensure they are valid for Open Liberty.`